salt/daemons/flo/zero.py

```diff
@@ -95,6 +95,7 @@ def run(self):
         except AttributeError:
             self.clients.setsockopt(zmq.SNDHWM, self.opts['rep_hwm'])
             self.clients.setsockopt(zmq.RCVHWM, self.opts['rep_hwm'])
+        self.clients.setsockopt(zmq.BACKLOG, self.opts.value.get('zmq_backlog', 1000))
```

Comment:
Let's remove the fallback of `1000` here and let it default to `None`. This will obviously stacktrace in the event that it's undefined but we want to fix the root cause of that issue. When we put fallbacks in like this, we have to chase them all down in the event that we want to change the default.